slack-moderate-words/filters.yaml

```diff
@@ -0,0 +1,4 @@
+- triggers:
+  - guys
+  action: chat.postEphemeral
+  message: "May I suggest \"all\" instead when addessing a group of people? Thank you. :slightly_smiling_face:"
```

Comment:
@mrbobbytables for help with the message text :)

Reply:
cc @kubernetes-sigs/wg-naming

Reply:
I can't really think of a better message at this time.

Reply:
@mrbobbytables thanks! sent an email to the wg-naming mailing list for review/ack - https://groups.google.com/g/kubernetes-wg-naming/c/d5JyKPUXA0g

Reply:
I'm with Bob. This is pretty good.

Reply:
@nikhita LGTM from the wg naming side.

/lgtm